src/structurizr/view/view.py

```diff
@@ -46,32 +46,29 @@ class ViewIO(BaseModel, ABC):
 
     key: str
     description: str
-    software_system_id: str = Field(default="", alias="softwareSystemId")
+    software_system_id: str = Field(default=None, alias="softwareSystemId")
     paper_size: Optional[PaperSize] = Field(default=None, alias="paperSize")
     automatic_layout: Optional[AutomaticLayoutIO] = Field(
         default=None, alias="automaticLayout"
     )
     title: str = ""
 
-    element_views: List[ElementViewIO] = Field(default=(), alias="elementViews")
+    element_views: List[ElementViewIO] = Field(default=[], alias="elements")
```

Comment:
Rename the field itself as well and remove the alias?

```suggestion
    elements: List[ElementViewIO] = Field(default=(), alias="elements")
```